api/v1/installation_types.go

```diff
@@ -126,6 +126,11 @@ type InstallationSpec struct {
 	// +optional
 	FlexVolumePath string `json:"flexVolumePath,omitempty"`
 
+	// VolumePlugin optionally specifies enablement of Calico CSI plugin. If not specified,
+	// VolumePlugin will be disabled by default. Valid values are "Enabled" or "Disabled".
+	// +optional
+	VolumePlugin string `json:"volumePlugin,omitempty"`
```

Comment:
I think we should have a type for this since it is two static strings possible. Then the CRD validation can handle ensuring it is a valid value.